101-azure-api-management-create/README.md

```diff
@@ -0,0 +1,3 @@
+# Azure API Management
+
```

Comment:
Please see the sample readme https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/sample-README.md